std-bits/table/src/main/java/org/enso/table/excel/xssfreader/XSSFReaderRow.java

```
@@ -0,0 +1,126 @@
+package org.enso.table.excel.xssfreader;
+
+import java.time.LocalDateTime;
+import java.util.SortedMap;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.DataFormatter;
+import org.enso.table.excel.ExcelRow;
+
+public class XSSFReaderRow implements ExcelRow {
+  private static final DataFormatter formatter = new DataFormatter();
+  private final SortedMap<Short, XSSFReaderSheetXMLHandler.CellValue> data;
+  private final boolean use1904Dates;
+
+  public XSSFReaderRow(
+      SortedMap<Short, XSSFReaderSheetXMLHandler.CellValue> data, boolean use1904Dates) {
+    this.data = data;
+    this.use1904Dates = use1904Dates;
+  }
+
+  @Override
+  public int getFirstColumn() {
+    return data.firstKey();
+  }
+
+  @Override
+  public int getLastColumn() {
+    return data.lastKey();
+  }
+
+  @Override
+  public Cell get(int column) {
+    // Not supported as we don't have the underlying Apache POI Cell object.
+    return null;
```

Could we throw an exception here to indicate that this path should not be used?